### PR TITLE
Dd 20 added reset password

### PR DESCRIPTION
**Pull request by YuraStork**

### CODE REVIEW

File: src/globals.d.ts [hunk 1]
```diff
@@ -1,4 +1,4 @@
 declare module 'public/assets/*.svg' {
-  const content: React.SVG
+  const content: (props: React.ComponentProps<'svg'>) => JSX.Element
   export default content
 }
```
Comment (Andrew01011997) [inline src/globals.d.ts]: `declare module '*.svg' {`
 `   import * as React from 'react';`
``
    const ReactComponent: React.FunctionComponent<React.SVGProps<SVGSVGElement>>;
``
`    export default ReactComponent;`
`}`

Could you use this one?

File: src/hooks/useAppNavigate.tsx [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+
+export const useAppNavigate = (hasNavigate = false, url: string, params: NavigateOptions = {}) => {
+  const navigate = useNavigate()
+
+  useEffect(() => {
+    if (hasNavigate) {
+      navigate(url, params)
+    }
+  }, [hasNavigate, navigate, url, params])
+}
```
Comment (Andrew01011997) [inline src/hooks/useAppNavigate.tsx]: could you return callback here and just call it when it needs?

File: src/hooks/useToastError.tsx [hunk 3]
```diff
@@ -3,10 +3,10 @@ import { toast } from 'react-toastify'
...
-export const useToastError = (isError: unknown) => {
+export const useToastError = (isError: unknown, message: string = ErrorMessages.OCCURED_ERROR) => {
   useEffect(() => {
     if (isError) {
-      toast.error(ErrorMessages.OCCURED_ERROR)
+      toast.error(message)
     }
-  }, [isError])
+  }, [isError, message])
 }
```
Comment (Andrew01011997) [inline src/hooks/useToastError.tsx]: If you use this hook you should already know that this is error, probably you need to change this int `useToastMessage` and indicate what type of notification should be shown

File: src/pages/disableGoogleAuthentificator/index.tsx [hunk 4]
```diff
@@ -35,23 +36,14 @@ export const DisableAuthentificator = () => {
   const { isUse2FA } = useSelector(userDataSelector)
   const navigate = useNavigate()
 
+  useAppNavigate(!isUse2FA, SETTINGS_URL)
```
Comment (Andrew01011997) [inline src/pages/disableGoogleAuthentificator/index.tsx]: does you meen useRedirect? this name will better understand what this hook does

File: src/pages/recoverPassword/content/index.tsx [hunk 5]
```diff
@@ -0,0 +1,72 @@
...
+export const RecoverPasswordContent = () => {
+  const [handleResetPassword, { isLoading, isSuccess, error, isError }] =
+    useRecoverPasswordMutation()
+  const navigate = useNavigate()
+
+  const formik = useFormik({
+    validationSchema: recoverPasswordValidationSchema,
+    onSubmit: handleResetPassword,
+    initialValues: {
+      email: ''
```
Comment (Andrew01011997) [inline src/pages/recoverPassword/content/index.tsx]: why you add initial value for email?

Comment (Andrew01011997) [reply src/pages/recoverPassword/content/index.tsx]: could it be initial undefined?

File: src/pages/recoverPassword/index.tsx [hunk 6]
```diff
@@ -0,0 +1,12 @@
+import { Container } from 'components/container'
+
+import { RecoverPasswordContent } from './content'
+import { RecoverPasswordWrapper } from './styles'
+
+export const RecoverPassword = () => (
+  <RecoverPasswordWrapper>
+    <Container>
```
Comment (Andrew01011997) [inline src/pages/recoverPassword/index.tsx]: could it be merged into 1 component?

**Review by Andrew01011997 — CHANGES_REQUESTED** [review]

File: src/hooks/useAppNavigate.tsx (hunk 2 above)
Comment (YuraStork) [reply src/hooks/useAppNavigate.tsx]: Потрібно будо створювати лишню перевірку

**Review by Andrew01011997 — APPROVED** [review]

**Pull request merged**